src/controllTools.js

```diff
@@ -0,0 +1,67 @@
+import { checkbox, editTask } from './index.js';
+import { getTask } from './data.js';
+
+export const deleteTask = () => {
+  const clearTask = document.querySelector('.clear-task');
+  clearTask.addEventListener('click', () => {
+    if (getTask().length > 0) {
+      const filterCompliteTask = getTask().filter(
+        (task) => task.completed !== true,
+      );
+      localStorage.setItem('Task-list', JSON.stringify(filterCompliteTask));
+      window.location.reload();
```

Comment:
what do you think about having a page reload to update the state of the task list on the page, is there any reason you wanted to do it this way? , I think doing this doesn't take advantage of the power of page dynamicity JavaScript provides to you by easily manipulating content of a page without reloading the page. using innerHTML or document append JS feature to dynamically update the content of the page could perform the same operation in a smoother and more optimized way of dynamically updating the page. without having to reload all resources.

Reply:
Thanks, I will change it and try to use `innerHTML` or `document.append` to dynamically update the content.